Cryptomator/AppDelegate.swift

```diff
@@ -78,7 +78,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		StoreObserver.shared.fallbackDelegate = coordinator
 		window = UIWindow(frame: UIScreen.main.bounds)
 		window?.tintColor = UIColor(named: "primary")
-		window?.rootViewController = navigationController
+		window?.rootViewController = coordinator?.rootViewController
```

Comment:
With this change, I suppose it's not necessary to create the `navigationController` here and provide it to the coordinator anymore. Maybe this could be refactored?